### DISCUSSION TITLE
How do I go about this error in my code for integrated analysis?

### DISCUSSION THREAD

**Discussion by pcitrano3** [satijalab/seurat#6157] [Q&A] [answered]
Hello. I am trying to run an integrated analysis on two sets of single-nuclei RNAseq data using the Seurat tutorial (), but I got an error when I ran the normalization and identification of variable features line of code. 

`# Loading required libraries`
`library(Seurat)`
`library(cowplot)`
`library(patchwork)`

`# Set up the Seurat Object`
`vgat.data <- Read10X(data.dir = "~/Desktop/VGAT Viral Data 1/")`
`vglut.data <- Read10X(data.dir = "~/Desktop/VGLUT3 Viral/")`

`# Initialize the Seurat object with the raw (non-normalized data)`
`vgat <- CreateSeuratObject(counts = vgat.data, project = "VGAT/VGLUT Integration", min.cells = 3, min.features = 200)`
`vglut <- CreateSeuratObject(counts = vglut.data, project = "VGAT/VGLUT Integration", min.cells = 3, min.features = 200)`

`# Merging the datasets
vgat <- AddMetaData(vgat, metadata = "VGAT", col.name = "Cell")`
`vglut <- AddMetaData(vglut, metadata = "VGLUT", col.name = "Cell")`
`merged <- merge(vgat, y = vglut, add.cell.ids = c("VGAT", "VGLUT"), project = "VGAT/VGLUT Integration")`

`# Split the dataset into a list of two seurat objects (vgat and vglut)`
`merged.list <- SplitObject(merged, split.by = "Cell")`

`# Normalize and Identify variable features for each dataset independently`
`merged.list <lapply(X = merged.list, FUN = function(x) {
  x <- NormalizeData(x)
  x <- FindVariableFeatures(x, selection.method = "vst", nFeatures = 2000)
})`

Error in merged.list < lapply(X = merged.list, FUN = function(x) { : 
  comparison of these types is not implemented

I am new to Seurat, so any explanation of why the error is popping up and how I can go around it would be greatly appreciated.

**Comment by samuel-marsh** [comment, marked as answer]
Hi,
Error is due to < symbol instead of assignment operator <- after merged.list.

Best,
Sam
